sympy/solvers/solveset.py

```diff
@@ -737,6 +858,18 @@ def solveset(f, symbol=None, domain=S.Complexes):
     >>> solveset(exp(x) > 1, x, R)
     (0, oo)
 
+    * If Non-symbol is passed :
```

Comment:
I would write it as something like "We can also solve the equation for a non symbol"